src/test/java/com/exasol/github/GitHubReleaseTest.java

```diff
@@ -0,0 +1,44 @@
+package com.exasol.github;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+class GitHubReleaseTest {
+    @Test
+    void testValidGitHubRelease() {
+        final GitHubRelease release = GitHubRelease.builder().version("1.0.0").header("header")
+                .releaseLetter("release letter").assets(Map.of("name", "path")).build();
+        assertAll(() -> assertThat(release.getVersion(), equalTo("1.0.0")),
+                () -> assertThat(release.getHeader(), equalTo("header")),
+                () -> assertThat(release.getReleaseLetter(), equalTo("release letter")),
+                () -> assertThat(release.getAssets(), equalTo(Map.of("name", "path"))));
+    }
+
+    @Test
+    void testGitHubReleaseEmptyVersion() {
```

Comment:
The three test are essentially white-box tests since they assume the order in which the parameters are tested.
A better test would be a parameterized test where you iterate over all combinations of either having a value or nil. This would also force you to collect all problems and list them in one error message, which helps users avoiding fix-run cycles.

Reply:
Please write a ticket for this.

Reply:
https://github.com/exasol/release-robot/issues/31